lib/tasks/factory_bot.rake

```diff
@@ -0,0 +1,13 @@
+namespace :factory_bot do
+  desc "Verify that all FactoryBot factories are valid"
```

Comment:
why do you need this task?

Reply:
I had some issues with `database_cleaner` and `factory_bot` and I tried this solution from [here](https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl).
Can I delete this file since the task was executed?

Reply:
If you are not using the task anymore I guess you can remove it.